test/conformance/tests/init.go

```diff
@@ -17,5 +18,10 @@ func init() {
 		operatorNamespace = "openshift-sriov-network-operator"
 	}
 
+	clusterType = os.Getenv("CLUSTER_TYPE")
```

Comment:
`clusterType` variable seems to be unused. Can we get rid of it?

Reply:
yep I didn't need to use it at the end